lib/templates/test-body-footer.html

```diff
@@ -36,8 +36,42 @@
     });
   }
 
+  function getModuleName (){
+    return jQuery.ajax({
+        url: '/get-module-name',
+        async: false
+    }).responseJSON // TODO: async after POC
+  }
+  function shouldExclude(moduleName){
+    return jQuery.ajax({
+      type: 'POST',      
+      url: '/should-exclude',
+      async: false,
+      data: {
+       moduleName: moduleName 
+      }
+    }).responseJSON
+  }
   if (typeof Testem !== "undefined" && Testem.afterTests) {
     Testem.afterTests(function(config, data, callback) {
+      var seen = {}
+      var appName = getModuleName().appName
+      for (var moduleName in requirejs.entries) {
+        if (typeof(seen[moduleName]) === 'undefined') {
+          var parts = moduleName.split('/')
+            try {
+              if (parts.length >= 1 && parts[0] === appName || parts.length >= 2 && parts[0] === 'dummy' && parts[1] === appName){
+                  if (!shouldExclude(moduleName).result){
```

Comment:
Can we flip this on its head: `POST` the result of `Object.keys(requirejs.entries)` and have the server return an array of modules to require?